### PR TITLE
Directions on how to setup custom build of images till PR gets merged

### PR DESCRIPTION
**Pull request by josmo**
Update to docs and .gitignore only

Pre-merge checklist:

* [n/a] Code changes have tests
* [n/a] Any changes to the config are documented in `docs/parameter_reference.md`
* [n/a] Any new _required_ config is documented in `README.md`
* [n/a] Any large changes have been verified by running a Drone job

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,3 +69,11 @@ drone-helm3 is largely backwards-compatible with drone-helm. There are some know
...
 
 Since helm 3 does not require Tiller, we also recommend switching to a service account with less-expansive permissions.
+
+### Contribution
+
+This repo is setup in a way that if you enable a personal drone server to build your fork it will
+ build and publish your image (makes it easier to test PRs and use the image till the contributions get merged)
+ 
+* Build local ```DRONE_REPO_OWNER=josmo DRONE_REPO_NAME=drone-ecs drone exec```
+* on your server (or cloud.drone.io) just make sure you have DOCKER_USERNAME, DOCKER_PASSWORD, and PLUGIN_REPO set as secrets
```
Comment (ErinCall) [inline README.md]: This is all in contributing.md already, isn't it?

File: .gitignore [hunk 2]
```diff
@@ -5,6 +5,8 @@
 *.so
 *.dylib
 
+.idea
```
Comment (ErinCall) [inline .gitignore]: What is a .idea file?

File: README.md (hunk 1 above)
Comment (josmo) [reply README.md]: just changed it to link to the contributing.md I didn't even know that was there

File: .gitignore (hunk 2 above)
Comment (josmo) [reply .gitignore]: intellij files that can be ignored

**Pull request merged**